### DISCUSSION TITLE
Are cache tags supported?

### DISCUSSION THREAD

**Discussion by DavidDeSloovere** [nitrojs/nitro#2695] [Q&A]
Who knows if cache tags are supported?
By that I don't mean a single key, but multiple tags. And later I would want to invalidate one or more tags.
I would like to achieve this pattern, with Nuxt on Vercel, but Vercel doesn't expose the data cache that Next uses currently, so I think I need a Nuxt/Nitro framework approach.
https://www.datocms.com/docs/content-delivery-api/cache-tags

From what I can tell by reading the docs, it's not directly possible. But maybe someone has an idea on how to achieve this?
https://nitro.unjs.io/guide/cache

**Comment by stafyniaksacha** [comment]
I wrote an article about cache invalidation in nitro. It may be outdated because it was written some month ago, but it may help you: https://cssninja.io/blog/server-cache-invalidation-in-nuxt-and-nitro

**Reply by DavidDeSloovere** [reply]
Although it's a very well written blog post, it does not offer a solution. The solution might just not be possible with nitro / unstorage.

A solution I currently have in mind is setting up a database to store the many-to-many relations between the nitro cache key and the cache tags from the CMS.
When I need to invalidate a specific cache tags from the CMS, I look up all the nitro cache keys, and remove those one by one.

**Reply by TheAlexLichter** [reply]
I personally can think that cache tags will be supported "natively" in the future. At the moment, a userland implementation is the way to go.
